### DISCUSSION TITLE
V2024.12.1

### DISCUSSION THREAD

**Discussion by WatsonYao** [WatsonYao/nian_app#121] [version]
●优化部分页面细节，修复部分用户反馈过来的问题，提升性能和稳定性。
年鉴新增单年样式（点击年份数字左右空白处可+或-）。

<hr /><em>This discussion was created from the release <a href='https://github.com/WatsonYao/nian_app/releases/tag/V2024.12.1'>V2024.12.1</a>.</em>

**Comment by cookie258** [comment]
为什么在这个版本有时会出现将相册里的图片放到nian里之后，永久删除了原相册的这张图片，而且也没有粘贴到nian的对应相册里。

**Reply by WatsonYao** [reply]
手机的系统最近是否有过升级？nian关于图片操作的代码几年都没有更新过了，估计是系统拦截了nian的复制或者粘贴操作。
一般在系统权限、安全隐私之类的设置里面查查看。

**Comment by soakup** [comment]
言语集上面的搜索不起作用了，点击搜索跳到了下方的输入框。

**Reply by WatsonYao** [reply]
言语集做的很简单，输入框输入后就是搜索了。

**Reply by soakup** [reply]
但是我输入框输入后没有筛选的结果，点输入框旁边的小宠物就是发出去一条新消息了
![Uploading IMG_20241219_225229.jpg…]()

**Comment by soakup** [comment]
刚才回复言语集反馈里图片好像有点显示问题，抱歉新发一条
![IMG_20241219_225229](https://github.com/user-attachments/assets/9aa8d39d-6e0d-4985-a5db-7b110c617dbf)

**Reply by WatsonYao** [reply]
不好意思，说错了，先在输入框输入要搜索的文字，再点击上面的搜索按钮（比较反直觉 哈哈哈

**Reply by soakup** [reply]
好的理解啦，谢谢🤣

**Comment by 1245655555** [comment]
请问为什么会无法拉起系统相册呢，无论是更换头像还是添加记本封面or背景都是点击后没有任何反应，卸载重下或者安装旧版本都是同样无反应

**Reply by WatsonYao** [reply]
![image](https://github.com/user-attachments/assets/330a73f1-68b3-4e57-979d-a2e15a3a2ef8)
